Traceback (most recent call last):
  File "/home/timothyprojectinfo/.local/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/timothyprojectinfo/.local/lib/python3.9/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/usr/local/lib/python3.9/dist-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
  File "/usr/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/home/timothyprojectinfo/.local/lib/python3.9/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/home/timothyprojectinfo/.local/lib/python3.9/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/timothyprojectinfo/.local/lib/python3.9/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import matplotlib.pyplot as plt  #plot library
fig, ax = plt.subplots(figsize=(5,5)) #you can adjust the figsize  (5,5)=(length,width)
        
budget_items = ['transportation','meal','presents','activities'] #categories
budget = [10,15,50,25] #enter the amounts for 'presents' and 'activities
total=sum(budget)
ax=plt.pie(budget,labels=budget_items,autopct=lambda p: '${:.0f}'.format(p * total / 100)) #make pie chart  autopct='%1.0f%%'
plt.gca().set_title("Christmas Budget") #add a title
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [0;32mIn[1], line 1[0m
[0;32m----> 1[0m [38;5;28;01mimport[39;00m [38;5;21;01mmatplotlib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mpyplot[39;00m [38;5;28;01mas[39;00m [38;5;21;01mplt[39;00m  [38;5;66;03m#plot library[39;00m
[1;32m      2[0m fig, ax [38;5;241m=[39m plt[38;5;241m.[39msubplots(figsize[38;5;241m=[39m([38;5;241m5[39m,[38;5;241m5[39m)) [38;5;66;03m#you can adjust the figsize  (5,5)=(length,width)[39;00m
[1;32m      4[0m budget_items [38;5;241m=[39m [[38;5;124m'[39m[38;5;124mtransportation[39m[38;5;124m'[39m,[38;5;124m'[39m[38;5;124mmeal[39m[38;5;124m'[39m,[38;5;124m'[39m[38;5;124mpresents[39m[38;5;124m'[39m,[38;5;124m'[39m[38;5;124mactivities[39m[38;5;124m'[39m] [38;5;66;03m#categories[39;00m

[0;31mModuleNotFoundError[0m: No module named 'matplotlib'

